concourse/pipeline.yml

```diff
@@ -148,6 +152,9 @@ jobs:
         file: git-master/concourse/tasks/smoke-test.yml
         timeout: 5m
         params:
-          # TODO: this should point at the production URL
           URL: 'https://govuk-coronavirus-find-support-prod.cloudapps.digital/urgent-medical-help'
-          MESSAGE: "Checks that the application deployed to production is not serving HTTP error codes. If this fails, you should investigate immediately."
+          MESSAGE: "Checks that the application deployed to production is not serving HTTP error codes"
+      - task: run-smoke-tests
+        file: git-master/concourse/tasks/run-smoke-tests.yml
```

Comment:
:thinking: so we're running this against both staging and prod...
I'm a little confused about what we'd expect to be stored in the postgres DB in the image and what might actually end up in staging / prod's database.

Why won't these tests incriment the production postgres counters when you hit the results page?
Isn't there a risk we're inflating real numbers every time we run the tests?

Reply:
In the business form smoke tests we don't write to the DB, and rely on user activity to alert us of failures:

https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/blob/master/app/controllers/coronavirus_form/check_answers_controller.rb#L12

Reply:
thanks, will add something similar